force-app/main/default/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js

```diff
@@ -361,11 +382,24 @@ export default class geFormWidgetTokenizeCard extends LightningElement {
         this.display.transitionTo('readOnly');
     }
 
+    isDataImportStatusImported() {
+        return this._widgetDataFromState[apiNameFor(DATA_IMPORT_STATUS)] === GIFT_STATUSES.IMPORTED
+    }
+
+    isPaymentStatusInReadOnlyMode() {
+        return this.isPaymentStatusAuthorized() || this.isPaymentStatusPending();
```

Comment:
It's a little confusing what we're trying to do with this function. If we are saying that AUTHORIZED and PEDNING are the two payment statuses that would render the widget in read-only mode, could we rename this to `isReadOnlyPaymentStatus()` ?

Reply:
I see further down that we already have a `readOnlyStatuses()` function as well. So what is this function's purpose?

Reply:
Oh nice, I hadn't seen that, I think this could be replaced by a call to `hasReadOnlyStatus` because yes, the goal here is to load the widget in read-only mode for certain statuses.